optimum/intel/openvino/quantization.py

```diff
@@ -157,7 +158,7 @@ def quantize(
         self,
         calibration_dataset: Dataset = None,
         save_directory: Union[str, Path] = None,
-        quantization_config: OVConfig = None,
+        ov_config: OVConfig = None,
```

Comment:
should we add a warning in `quantize` that `quantization_config` is deprecated like

```
quantization_config = kwargs.pop("quantization_config", None)
if quantization_config is not None:
    logger.warning("The argument `quantization_config` is deprecated, and will be removed in optimum-intel v1.6.0, please use `ov_config` instead"")
ov_config = ov_config or quantization_config
```

Reply:
Done